### PR TITLE
feat: Allow to take screenshots of multiple Android displays

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Similar to https://github.com/appium/appium-mac2-driver#macos-screenshots

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@KazuCocoa Could you please help to test this feature on a real device with multiple screens?

**Comment by KazuCocoa** [comment]
of course. I haven't tried such multiple screens, so let me configure it and take a look. (I have a display so maybe i can configure the env)

**Comment by KazuCocoa** [comment]
the implementation itself seems reasonable, but still I cannot find any apps which can work with multiple display (not cast)...

File: lib/commands/screenshot.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
+import _ from 'lodash';
+import B from 'bluebird';
+
 const commands = {};
 
+// Display 4619827259835644672 (HWC display 0): port=0 pnpId=GGL displayName="EMU_display_0"
+const DISPLAY_PATTERN = /^Display\s+(\d+)\s+\(.+display\s+(\d+)\).+displayName="([^"]+)/gm;
```
Comment (KazuCocoa) [inline lib/commands/screenshot.js:7]: I met an empty string case as below:
```
Display 4630947150978042497 (HWC display 0): port=129 pnpId=QCM displayName=""
```
may need to tune the patter a bit

Comment (KazuCocoa) [reply lib/commands/screenshot.js:7]: after the regex tweak: (only have one display case tho)
```
$ driver.execute_script 'mobile:screenshots'
=> {"4630947150978042497"=>
  {"id"=>"4630947150978042497",
   "isDefault"=>true,
   "name"=>"",
   "payload"=>
    "iVBORw0KGgoAAAANSUhEUgAABDgAAAjoCAYAAAA5jhrUAAABJGlDQ1BTa2lhAAAokX2RPUvDUBSGH4uLUkHRwcEhYxfFD+iiLlUIOkms0OqUpmkUmhiSlOrs6o9wc3YUoXMXQRCcxN8gDq71TTskUOJ7Ofc+9+Xcj3MvlFaRSovgB0lkmTWj0Tw3yMl24pBizcDvR9rD+/o/eUWaa7uxo/FbkUQ6XFu2xSvehG9Sbk34PuWobh2IH8QVL8etHPeTMBEP03wnjFL+FO/53Z6T3ZuyG5ydamwo1oixMKlhUCfCJpDTwRUb9Lki4VJ0KIoJ6SrjVvMTdtSbmvn0lDOl7D3vvmB/MBqNXjLveABPVZh/zrzKLiyVYfga2pE9tmYVpU4Hfh5hoQnLb1pzMX1G9hcFtRrjWk2u1TxV4co5Uq0OG6JtNtmi+geOy02O5f5fawAAAARzQklUCAgICHwIZIgAACAASURBVHic7N13fFRV+sfx75T0QBJI6IQiEFBAUKQoYlkVEZRuB9dCQERZQQK6UkSwIPhbioAdUQMKiiDgglgpIiA2qtJRuiEQAmkz9/cHzuxMZtInyQz5vPeVl5lbzj3nzmSZ89xznmNqWL+BIQAAAAAAgABmLu8KAAAAAAAAlBQBDgAAAAAAEPAIcAAAAAAAgIBnlUjBAQAAAAAAAhsjOAAAAAAAQMCzMn4DAAAAAAAEOkZwAAAAAACAgGc1mUzlXQcAAAAAAIASYQQHAAAAAAAIeAQ4AAAAAABAwLOySiwAAAAAAAh0jOAAAAAAAAABjwAHAAAAAAAIeAQ4AAAAAABAwCPAAQAAAAAAAp7VIMsoAAAAAAAIcIzgAAAAAAAAAY8ABwAAAAAACHgEOAAAAAAAQMAjwAEAAAAAAAKe1WQylXcdAAAAAAAASoQRHAAAAAAAIOAR4AAAAAAAAAHPWt4VAICSMpvNCgkJUaVKlRRXrZoqV6qksLAwmUwm2Ww2ncvIUOrJkzp27JjOnj2rrKwsGYZR3tX2ibCwMFWNjVX1atVUpWpVWcxmZWZm6sSJEzp8+LBSU1OVk5NT3tUEAAAASp1VF8Z3fCBfJpNJISEhCg0NlSPvjCEp49w5ZWdny2azlW8FS0FFaXOtWrXUrl073XjTTWrWrJkqV66k0LAwBQUFSZIMw1BmZqYyMzN18uRJbf5hs7744gtt3LhRqSdPlnPti89iseiyyy7THXfeoQ5XXqlKfwd1zGazcnJylJ6ermNHj+mzz5br448X6Y+DBy+YoA4AAKg4oqKi1KRJE1WrVk1VqlZR5cqVdfr0aaX8laJjx47pt99+06lTp8q7mnnq3aePateune8x06ZOLaPaXPhMDes34BsvLggmk0lhYWGKiYnRsWPHlJ2dLavVqujoaMVUiVGzZs3UtGlThYdHSJIyMjO0dcsWHT1yVIcOHdLJkyeVnp7utRx/VRHbLJ1vd934urr11lvV5ZZb1KRJE1kslkKff/bsWe3YvkMfffSRPl+5Un/99Vcp1tb3oqKiNCAxUb1691L16tXzPdZut+v333/XrJkz9emST8uohgAAAMXTvEVzXXnlVerQob2aNmumuLg4t/0mk8njoc3x48e1Y/t2fffdeq1du1Zbt2wpyyrnK3n+PLVr1y7fYy5q0LCManPhI8CBC4LJZFJUdLRuvvlm3XXXXXpp0iTt3LlTjZs0Vs+ePdXpmmsUGxvr9VybzaZdu3ZpXnKyvv7qa2VnZ+va667TnXfdqUkvvqiNGzb6ZYfftc133nWnJk96ydnmXr166epOnYrc5rvuuksvvviC37ZZOt/ua6+7Vo8//riaXXyxzGb3VEKOf/AMw5DdbpfZbHaOYMm9alROTo7WrVuniRMmatfvv5dNA3zg+Rde0O133F6kc86cOaMJzz6rBR8uKKVaAQAAFE/dunXVu09v9ezVyznaoTirfTq+B/7555/6ZNEiLVywUAcPHvRpXYuKAEfZMjWoX58Ah4+FhYWpadOmqlOnjoKCgnTo0CH99ttvSklJKZPrX96mjWrVqqXIiAidOnVKW7Zu1YH9+8vk2uUlPDxc3W69VaNHj1ZYWJgOHz6sb7/9VrfddpsiIiLcjnV0fHM/8bfZbNqzZ482bdqk2267TWFhYdqzZ4+eevJJbdq0qSybUygVtc33P/CA+vfvr9jYWOc/fDabTT///LO+/eYbbd++XX/++adOnTqlnJwcBQcHKyYmRvXq1VPzFi10ww03qF69es7AiM1m08GDBzVt6lQtXrzYr6dxBAUF6fFhw3T//fcrJCSkyOenp6dr4MCBWrd2bSnUDgAAoGhatGihIY8+qhtuuKFYAY2
```

Comment (mykola-mokhnach) [reply lib/commands/screenshot.js:7]: thanks, I've updated the regexp

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**